src/Microsoft.DotNet.Darc/Darc/Models/PopUps/UpdateSubscriptionPopUp.cs

```diff
@@ -32,7 +32,14 @@ public class UpdateSubscriptionPopUp : SubscriptionPopUp
             header: [
                 new Line($"Use this form to update the values of subscription '{subscription.Id}'.", true),
                 new Line($"Note that if you are setting 'Is batchable' to true you need to remove all Merge Policies.", true),
-                new Line()
+                Line.Empty,
+                new("Source and target directories only apply to source-enabled subscription (VMR code flow subscriptions).", true),
+                new("They define which directory of the VMR (under src/) are the sources synchronized with.", true),
+                new("Only one of those needs to be set based on whether the source or the target repo is the VMR.", true),
```

Comment:
```suggestion
                new("Only one of those can be set based on whether the source or the target repo is the VMR.", true),
```

Reply:
I think needs to, because one must be set

Reply:
when I'm reading this, `needs to be` means that 1 needs to be set, but you can also set the other one. Perhaps `must be` set is better?

Reply:
```suggestion
                new("Only one of those must be set based on whether the source or the target repo is the VMR.", true),
```

Reply:
I put `must` but `needs to` vs `must` is almost the same?